lib/puppet/type/firewall.rb

```diff
@@ -840,6 +840,10 @@ def should_to_s(value)
     end
   end
 
+  newproperty(:ctdir) do
+    newvalues(:REPLY, :ORIGINAL)
```

Comment:
Preferably there should be a brief `desc` here

Reply:
I added a desc